work8/index.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+const is_false = false;
+
+function getObj() {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      if (is_false === false) {
+        reject("Error");
+      }
+      else {
+        resolve([
+          {
+            to: "bookmark.html",
+            img: "1.png",
+            alt: "画像1",
+            text: "ブックマーク",
+          },
+          {
+            to: "message.html",
+            img: "2.png",
+            alt: "画像2",
+            text: "メッセージ"
+          },
+        ]);
+      }
+    }, 3000);
+  });
+}
+
+getObj().then((items) => {
+  const ul = document.getElementById("js-list");
+  const fragment = document.createDocumentFragment();
+  items.forEach((item) => {
+    const li = document.createElement("li");
+    const a = document.createElement("a");
+    const img = document.createElement("img");
+
+    a.href = item.to;
+    a.textContent = item.text;
+    img.src = item.img;
+    img.alt = item.alt;
+
+    fragment.appendChild(li).appendChild(a).prepend(img);
+  });
+  ul.style.backgroundImage = "none";
+  ul.appendChild(fragment);
+})
+  .catch((error) => {
+    console.log(error)
+  })
```

Comment:
```suggestion
  .catch((error) => {
    console.error(error)
  })
```
`console.log`はログをコンソールに出力するものです。
この問題は`catch節でエラーをキャッチ`することが目的なのでコンソールにエラーを出力させる`console.error`が正しいです。
[参考記事](https://lab.syncer.jp/Web/API_Interface/Reference/IDL/console/error/#:~:text=console.error()%E3%81%AF%E3%80%81%E3%82%B3%E3%83%B3%E3%82%BD%E3%83%BC%E3%83%AB,%E3%83%AD%E3%82%B0%E3%82%92%E5%87%BA%E5%8A%9B%E3%81%97%E3%81%BE%E3%81%99%E3%80%82)

Reply:
大変遅くなりました。
少し、調べてみました。色々なconsoleがありそれぞれに適したメソッドがあるようですね…

とても勉強になりました。ありがとうございます。
308e457